packages/studio-ui/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@yext/studio-ui",
+  "version": "0.22.0",
+  "types": "./lib/src/index.d.ts",
+  "main": "./lib/src/index.js",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./lib/src/index.d.ts",
+      "import": "./lib/src/index.js",
+      "require": "./lib/src/index.js"
+    }
+  },
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "dev": "npm run copy-svg && tsc --watch --preserveWatchOutput -p tsconfig.json",
+    "copy-svg": "npx copyfiles \"src/**/*.svg\" lib",
+    "build": "rimraf lib && npm run copy-svg && tsc -p tsconfig.json",
+    "test": "jest",
+    "typecheck-jest": "npx tsc -p tests/tsconfig.json"
+  },
+  "dependencies": {
+    "@dhmk/zustand-lens": "^2.0.5",
+    "@minoru/react-dnd-treeview": "^3.4.1",
+    "@restart/ui": "^1.5.2",
+    "@vitejs/plugin-react": "^4.0.4",
+    "autoprefixer": "^10.4.14",
+    "cac": "^6.7.14",
+    "classnames": "^2.3.2",
+    "cross-env": "^7.0.3",
+    "immer": "^9.0.21",
+    "lodash": "^4.17.21",
+    "path-browserify": "^1.0.1",
+    "postcss": "^8.4.27",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-modal": "3.16.1",
+    "react-toastify": "^9.1.1",
+    "react-tooltip": "^5.18.0",
+    "tailwind-merge": "^1.8.1",
+    "tailwindcss": "^3.3.3",
+    "vite": "^4.4.7",
+    "vite-plugin-svgr": "^2.4.0",
```

Comment:
`studio-ui` doesn't use Vite, does it?

Reply:
it will need to be added back in the following pr